src/application/port/input/vector_db.py

```diff
@@ -0,0 +1,9 @@
+class VectorInputPort:
+    def store_vector(self, vector_data):
```

Comment:
i think we should handle vector list[float] separately from vector metadata (like text chunk, doc name and other params)